pfioh/pfioh.py

```diff
@@ -173,84 +178,24 @@ def do_GET_withCompression(self, d_msg):
             b_zip           = True
             # str_archive    = 'zip'
 
-        # If specified (or if the target is a directory), create zip archive
-        # of the local path
-        if b_zip:
-            self.qprint("Zipping target '%s'..." % str_serverPath, comms = 'status')
-            d_fio   = zip_process(
-                action  = 'zip',
-                path    = str_serverPath,
-                arcroot = str_serverPath
-            )
-            d_ret['zip']        = d_fio
-            d_ret['status']     = d_fio['status']
-            d_ret['msg']        = d_fio['msg']
-            d_ret['timestamp']  = '%s' % datetime.datetime.now()
-            if not d_ret['status']:
-                self.qprint("An error occurred during the zip operation:\n%s" % d_ret['stdout'],
-                            comms = 'error')
-                self.ret_client(d_ret)
-                return d_ret
+        if b_swiftStorage:
```

Comment:
At a min this should be abstracted behind a method.  Would a statically initialized instance make more sense though?